Update examples in docstring to use TF 2.x code

yongtang opened the pull request:
The examples in docstrings of two APIs, tf.histogram_fixed_width_bins
and tf.histogram_fixed_width still used TF 1.x code.

This PR updates the docstring to use TF 2.x code in examples.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

yongtang committed:
Update examples in docstring to use TF 2.x code

The examples in docstrings of two APIs, tf.histogram_fixed_width_bins
and tf.histogram_fixed_width still used TF 1.x code.

This PR updates the docstring to use TF 2.x code in examples.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

alextp reviewed (commented):
Please refer to https://www.tensorflow.org/community/contribute/docs_ref and make these things doctests

yongtang committed:
Update tf.histogram_fixed_width docstring to comform to Python doctest

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

yongtang commented:
Thanks @alextp for the review, the PR has been updated to conform to python doctest. Please take a look and let me know if there are any issues.

yongtang committed:
Remove `print` in docstring as it causes discrepancy in doctest

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

gbaned commented:
@yongtang Can you please address Ubuntu Sanity errors? Thanks!

yongtang committed:
Replace print(hist) with hist.numpy() to avoid print call, also avoid pylint too long (80) error

Remove extra "```" as doctest use ">>>" and "..." instead, and fix pylint issue by shorten the output

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

alextp commented on tensorflow/python/ops/histogram_ops.py:
```diff
@@ -62,17 +62,14 @@ def histogram_fixed_width_bins(values,
+  >>> value_range = [0.0, 5.0]
+  >>> new_values = [-1.0, 0.0, 1.5, 2.0, 5.0, 15]
+  >>> indices = tf.histogram_fixed_width_bins(new_values, value_range, nbins=5)
+  >>> print(indices)
```
Apparently our internal linter is stupid and doesn't like print in comments (it warns about debug prints). Thankfully you can just replace this with `indices` and it'll work just the same.

Can you fix here and in the other print statements?

yongtang commented:
Thanks @alextp for the help. I changed to `indices.numpy()` as `indices` directly will output more than 80 and pylint may fail (line-too-long). Let me know if this works.
